cabal.project

```diff
@@ -38,3 +38,18 @@ import: ./asserts.cabal
 if(os(windows))
   constraints:
     bitvec -simd
+
+allow-newer:
```

Comment:
Couldn't we wrap this in a conditional for the GHC version?

Reply:
I hope to remove these before  pull it out of draft mode.

Reply:
It is some time before the `ledger` packages are released to CHaP so I have made them conditional on GHC version.